e2e/content-services/version/version-actions.e2e.ts

```diff
@@ -157,7 +157,7 @@ describe('Version component actions', () => {
 
         await BrowserActions.click(versionManagePage.showNewVersionButton);
 
-        await browser.executeScript(' setTimeout(() => {document.querySelector("div[data-automation-id=\'cancel-upload-progress\']").click();}, 1000)');
+        await browser.executeScript(' setInterval(() => {document.querySelector("div[data-automation-id=\'cancel-upload-progress\']").click();}, 1000)');
```

Comment:
interval is not going to stop and will be clicking every 1000 ms forever

Reply:
it should be ok , that pice of code dye after the end of this it('